### PR TITLE
add newsletter no. 3 as blog post

### PR DESCRIPTION
**Pull request by argshook**

### CODE REVIEW

File: blog/2021-03-02-newsletter-3.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+title: Newsletter Mar 2021 - Out-of-order Inserts
+author: QuestDB
+author_url: https://github.com/questdb
+author_image_url: https://avatars.githubusercontent.com/u/52297642?s=200&v=4
+description:
+  We're happy to share the latest updates about QuestDB on the web while we work
+  on bringing version 6.0.0 to fruition, and we hope you enjoy the latest
+  content from recent weeks.
+tags: [newsletter]
+image: /img/blog/newsletter-preview.png
```
Comment (amyshwang) [inline blog/2021-03-02-newsletter-3.md:11]: Not showing.

File: blog/2021-03-02-newsletter-3.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+slug: newsletter-out-of-order-inserts
+---
+
+import Banner from "@theme/Banner"
+import Screenshot from "@theme/Screenshot"
+
+<Banner
+  alt="Image of QuestDB logo"
+  src="/img/blog/newsletter.png"
```
Comment (amyshwang) [inline blog/2021-03-02-newsletter-3.md:20]: Not rendering - although I updated the PR with master...

**Review by amyshwang — CHANGES_REQUESTED** [review]
Preview and banner images not working.

File: blog/2021-03-02-newsletter-3.md (hunk 1 above)
Comment (argshook) [reply blog/2021-03-02-newsletter-3.md:11]: oops, clicked `resolve` accidentally.

although these images do load for me ok it seems.
i'm looking at netlify preview [here in the blog posts list](https://deploy-preview-1262--unruffled-blackwell-31bfb2.netlify.app/blog/tags/newsletter/) and [here on the blog post itself](https://deploy-preview-1262--unruffled-blackwell-31bfb2.netlify.app/blog/newsletter-out-of-order-inserts/)

could it be you checked it before preview got updated?

File: blog/2021-03-02-newsletter-3.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+author: QuestDB
+author_url: https://github.com/questdb
+author_image_url: https://avatars.githubusercontent.com/u/52297642?s=200&v=4
+description:
+  We're happy to share the latest updates about QuestDB on the web while we work
+  on bringing version 6.0.0 to fruition, and we hope you enjoy the latest
+  content from recent weeks.
+tags: [newsletter]
+image: /img/blog/newsletter-preview.png
+slug: newsletter-out-of-order-inserts
```
Comment (amyshwang) [inline blog/2021-03-02-newsletter-3.md:12]:
```suggestion
slug: newsletter-release-out-of-order-ingestion-capacity-planning
```

**Review by amyshwang — APPROVED** [review]

**Pull request merged**